projects/ccf-rui/src/app/shared/components/visibility-menu/visibility-menu.component.ts

```diff
@@ -40,12 +50,28 @@ export class VisibilityMenuComponent {
    */
   @Output() readonly itemsChange = new EventEmitter<VisibilityItem[]>();
 
+  /**
+   * slider component
+   */
+  @ViewChild('slider', { static: false }) slider: OpacitySliderComponent;
+
   /**
    * Creates an instance of visibility menu component.
    *
+   * @param el document element
    * @param ga Analytics service
    */
-  constructor(private readonly ga: GoogleAnalyticsService) { }
+  constructor(el: ElementRef<Element>, private readonly ga: GoogleAnalyticsService) {
+    el.nativeElement.addEventListener('mousedown', () => {
+      const input = el.nativeElement.querySelector('.visible .opacity-slider');
+      if (input) {
+        input.addEventListener('input', (event: InputEvent) => {
+          const target = event.target as HTMLInputElement;
+          this.slider.changeOpacity(target.value);
```

Comment:
This seems a bit odd. Shouldn't the `ccf-opacity-slider` manage updates to itself?